src-backbone/app/js/routers/auth/auth_controller.js

```diff
@@ -1,42 +1,44 @@
-var Helpers = require('utils/helpers');
-
 module.exports = Marionette.Controller.extend({
 
+    constructor : function (options) {
+        this.app = options && options.app;
```

Comment:
technically for here and all other instances of `this.app = options && options.app;` I can skip the null check on `options` to enforce the presence of `app` passed into the constructor.